README.md

```diff
@@ -50,13 +50,33 @@ git clone https://github.com/Start9Labs/documentation.git && cd documentation
 git submodule update --init
 ```
 
+1. Prep NVM:
+
+```
+nvm use v16.11 || nvm install v16.11 && nvm use v16.11
+```
+
+1. Ensure poetry is updated
+
+```
+poetry update
```

Comment:
I'm guessing this is not accomplished by the poetry install in the setup script?

Reply:
I don't think so, or at least it doesn't obviously, and certainly not explicitly (as can be seen here):

https://install.python-poetry.org/